<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSMM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainArea">
      <item>
       <widget class="MapDescriptorWidget" name="tableWidget">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="autoScrollMargin">
         <number>8</number>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>71</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="modListWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="toolbar" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="mapToolbar" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="addMap">
            <property name="text">
             <string>Add Map</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="duplicateMap">
            <property name="text">
             <string>Duplicate Map(s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeMap">
            <property name="text">
             <string>Remove Map(s)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="quickSetup">
         <property name="text">
          <string>Quick Setup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importModPack">
         <property name="text">
          <string>Import Mod List</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_WBFS_ISO"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExport_to_WBFS_ISO"/>
    <addaction name="actionExport_to_Folder"/>
    <addaction name="actionExport_to_Riivolution"/>
    <addaction name="actionShow_CSMM_Network_Cache_In_File_System"/>
    <addaction name="actionOpen_Log_Folder"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Optional Patches</string>
    </property>
    <addaction name="actionPatch_MarkerCode"/>
    <addaction name="actionSeparate_Save_Game"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCSMM_Help"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionItast_csmm_brsar"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_map_list_csv"/>
    <addaction name="actionSave_map_list_csv"/>
    <addaction name="separator"/>
    <addaction name="actionValidate"/>
    <addaction name="separator"/>
    <addaction name="actionExport_default_modlists_txt"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCSMM_Help">
   <property name="text">
    <string>CSMM Help</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Extracted Game Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_to_Folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export to Extracted Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionImport_WBFS_ISO">
   <property name="text">
    <string>Import WBFS/ISO</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExport_to_WBFS_ISO">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export to WBFS/ISO</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Re_Download_External_Tools">
   <property name="text">
    <string>(Re-)Download External Tools</string>
   </property>
  </action>
  <action name="actionValidate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Validate</string>
   </property>
  </action>
  <action name="actionPatch_MarkerCode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Patch Marker Code (marker_code=02)</string>
   </property>
  </action>
  <action name="actionItast_csmm_brsar">
   <property name="text">
    <string>Save clean Itast.csmm.brsar</string>
   </property>
  </action>
  <action name="actionLoad_map_list_csv">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load map list (*.csv / *.yaml)</string>
   </property>
  </action>
  <action name="actionSave_map_list_csv">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save map list (*.yaml)</string>
   </property>
  </action>
  <action name="actionExport_default_modlists_txt">
   <property name="text">
    <string>Export default modlist.txt</string>
   </property>
  </action>
  <action name="actionAppend_Author_to_Map_Description">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Append Author to Map Description</string>
   </property>
  </action>
  <action name="actionShow_CSMM_Network_Cache_In_File_System">
   <property name="text">
    <string>Show CSMM Network Cache In File System</string>
   </property>
  </action>
  <action name="actionExport_to_Riivolution">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export to Riivolution</string>
   </property>
  </action>
  <action name="actionOpen_Log_Folder">
   <property name="text">
    <string>Open Log Folder</string>
   </property>
  </action>
  <action name="actionSeparate_Save_Game">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Separate Save Game</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapDescriptorWidget</class>
   <extends>QTableWidget</extends>
   <header>mapdescriptorwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
